src/os/ui/ol/interaction/mousewheelzoominteraction.js

```diff
@@ -1,105 +1,43 @@
- * @extends {ol.interaction.MouseWheelZoom}
+ * Adjusts the OpenLayers mouse wheel zoom interaction for OpenSphere.
  * @param {olx.interaction.MouseWheelZoomOptions=} opt_options Options.
- */
-os.ui.ol.interaction.MouseWheelZoom = function(opt_options) {
-  os.ui.ol.interaction.MouseWheelZoom.base(this, 'constructor', opt_options);
-};
-goog.inherits(os.ui.ol.interaction.MouseWheelZoom, ol.interaction.MouseWheelZoom);
-
-
-/**
- * Override to reduce {@link ol.MOUSEWHEELZOOM_TIMEOUT_DURATION} so it doesn't feel so sluggish. It may still feel
- * insane on OSX...
- *
- * @param {ol.MapBrowserEvent} mapBrowserEvent Map browser event.
- * @return {boolean} `false` to stop event propagation.
- * @this ol.interaction.MouseWheelZoom
+ * @extends {ol.interaction.MouseWheelZoom}
+ * @constructor
  *
- * @suppress {accessControls|duplicate} Replacing a static function from OL3.
+ * @suppress {accessControls} To allow changing private parameters lacking a corresponding options value.
  */
-ol.interaction.MouseWheelZoom.handleEvent = function(mapBrowserEvent) {
-  var stopEvent = false;
-  if (mapBrowserEvent.type == ol.events.EventType.WHEEL || mapBrowserEvent.type == ol.events.EventType.MOUSEWHEEL) {
-    // only handle the event if no modifier keys are held down, but still block the event if they are. this will prevent
-    // accidentally zooming the browser when the event is handled by the canvas.
-    if (ol.events.condition.noModifierKeys(mapBrowserEvent)) {
-      var map = mapBrowserEvent.map;
-      var wheelEvent = /** @type {WheelEvent} */ (mapBrowserEvent.originalEvent);
-
-      if (this.useAnchor_) {
-        this.lastAnchor_ = mapBrowserEvent.coordinate;
-      }
-
-      // Delta normalisation inspired by
-      // https://github.com/mapbox/mapbox-gl-js/blob/001c7b9/js/ui/handler/scroll_zoom.js
-      // TODO There's more good stuff in there for inspiration to improve this interaction.
-      var delta;
-      if (mapBrowserEvent.type == ol.events.EventType.WHEEL) {
-        // treat horizontal scroll the same as normal scroll
-        delta = wheelEvent.deltaY || wheelEvent.deltaX;
-
-        if (ol.has.FIREFOX &&
-            wheelEvent.deltaMode === goog.global.WheelEvent.DOM_DELTA_PIXEL) {
-          delta /= ol.has.DEVICE_PIXEL_RATIO;
-        }
-        if (wheelEvent.deltaMode === goog.global.WheelEvent.DOM_DELTA_LINE) {
-          delta *= 40;
-        }
-      } else if (mapBrowserEvent.type == ol.events.EventType.MOUSEWHEEL) {
-        delta = -wheelEvent.wheelDeltaY;
-        if (ol.has.SAFARI) {
-          delta /= 3;
-        }
-      }
-
-      this.delta_ += delta;
+os.ui.ol.interaction.MouseWheelZoom = function(opt_options) {
+  var options = opt_options || {};
 
-      if (this.startTime_ === undefined) {
-        this.startTime_ = Date.now();
-      }
+  // change the default zoom timeout period so it feels less sluggish
+  options.timeout = options.timeout != null ? options.timeout : 10;
 
-      var duration = 10;
```

Comment:
This function was previously replaced to change the timeout interval, which is now configurable.